src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -2276,6 +2276,171 @@ class StubGenerator: public StubCodeGenerator {
     StubRoutines::_arrayof_jint_fill = generate_fill(T_INT, true, "arrayof_jint_fill");
   }
 
+  void generate_aes_loadkeys(const Register &key, VectorRegister *working_vregs, int reg_number) {
```

Comment:
Hi, maybe `rounds` is more readable than `reg_number` here.

Reply:
Renamed